cmd/juju/application/config.go

```diff
@@ -94,6 +94,24 @@ settings back to their default value as defined in the charm metadata:
 
 juju config apache2 --reset servername
 juju config apache2 --reset servername,lb_balancer_timeout
+`
+
+	examples = `
+To view all configuration values for an application, run
+
+    juju config mysql --format json
+
+To set a configuration value for an application, run
+
+    juju config mysql foo=bar
+
+To set multiple configuration values for an application, use --reset to reset one or more keys:
```

Comment:
PS A small comment, but maybe you should remove the ", run" and just use a ":". E.g., instead of "To set a configuration value for an application, run" use "To set a configuration value for an application:". It would be more consistent with what we have everywhere else.

Reply:
I see this has been marked as resolved but there don't seem to be any changes.